notifications/src/main/java/org/openedx/notifications/presentation/inbox/NotificationsInboxFragment.kt

```diff
@@ -213,7 +230,7 @@ private fun InboxView(
                                 }
                             }
 
-                            if (scrollState.shouldLoadMore(firstVisibleIndex, 4)) {
+                            if (scrollState.shouldLoadMore(4)) {
```

Comment:
better to move this value to a const

Reply:
We haven't used a constant for this throughout the app, so let's keep it as it is here too.